libs/akita/src/lib/store.ts

```diff
@@ -176,7 +184,12 @@ export class Store<S = any> {
     }
 
     if (!this.store) {
-      this.store = new BehaviorSubject(this.storeValue);
+      this.store = new BehaviorSubject({ state: this.storeValue });
+      this.store.subscribe(({ action }) => {
```

Comment:
We need to run it only in dev-mode.

Reply:
Will now check if `isDev()` is true and only then subscribe to the store to dispatch logs.